Docs: Card section paredown (#1136)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/card-section.mdx

import ComponentDescription from "components/ComponentDescription";
import ComponentFooter from "components/ComponentFooter";
import sketchSymbol from '../../images/sketch-symbol.svg';
import githubSymbol from '../../images/icon/github-icon.svg';
import reactSymbol from '../../images/react-icon.svg';
import webComponentsSymbol from '../../images/web-components-icon.svg';
import ResourceLinks from "components/ResourceLinks";

<ComponentDescription name="Card section" type="layout" />

<AnchorLinks>
<AnchorLink>Overview</AnchorLink>
<AnchorLink>Card section</AnchorLink>
<AnchorLink>Card section with images</AnchorLink>
<AnchorLink>Feedback</AnchorLink>
</AnchorLinks>

## Overview

> Card section is a collection of text-based cards that can include images instead of body copy.

Reviewer: ```suggestion
Card section and Card section with images are section-level components. The left side container of each includes a section heading, while the right side includes an array of cards, either with body copy (Card section) or images (Card section with images). 
```

## Card section

> Card section uses uses cards with a header and body copy to present links to other parts of IBM.com. 

Reviewer: ```suggestion
Card section uses cards with a header and body copy to present links to other pages within IBM.com. 
```

![A visual of Card section simple](../../images/pattern/card-section/card-section-simple.jpg)
<Caption>An example of Card section</Caption>

### Modifiers

#### With CTA

> In this variation, the last card in the Card group is set to high contrast and can be used to highlight the primary action that the user may take.

Reviewer: ```suggestion
In this variation, the last card in the Card group is set to high contrast and can be used to highlight the primary action that the user can take.
```

![A visual of Card section with CTA](../../images/pattern/card-section/card-section-cta.jpg)

> <Caption>When Card section has a CTA, the last card will be set to high contrast</Caption>

Reviewer: ```suggestion
<Caption>When Card section has a CTA, the last card will be set to high contrast.</Caption>
```

> <ResourceLinks name="Card section" type="layout" />

Reviewer: @RichKummer, you'll need to add the multi prop to the `Resources` for `Card section` because it's currently showing as an H2 and it doesn't have the full heading name.

> ### Content guidance

Reviewer: ```suggestion
### Content guidance for Card section
```

> Note: This uses the underlying component "Card group". You can find more information about that component <a href="https://www.ibm.com/standards/carbon/components/card-group">here</a>.

Reviewer: ```suggestion
 Note: This uses the underlying component [Card group](/components/card-group).
```

For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

## Card section with images

> Card section with images brings images into each card in the section. Like Card section, this components is typically used for presenting resources or links, but also presents images in each card. When using Card section with media, avoid using a CTA card as this will cause the card heading to misalign.

Reviewer: ```suggestion
Card section with images brings images into each card in the section. Like Card section, this component is  used for presenting resources or links, but presents images instead of body copy in each card. 

When using this component, avoid using a CTA card as this will cause the card heading to misalign.
```

> To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.

Reviewer: ```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).
```

## Card section with images

> Card section with images brings images into each card in the section. Like Card section, this components is typically used for presenting resources or links, but also presents images in each card.

Reviewer: ```suggestion
Card section with images includes images in each card. Like Card section, this component is used for presenting resources or links, but adds visual interest with images included. 
```

![Card section images](../../images/pattern/card-section/card-section-images.jpg)
<Caption>An example of Card section with images</Caption>

> ### Resources for Card section with images

Reviewer: Let's change this to the Resource link with the multi prop included, so that we're automated. 🤖

<CardGroup>
  <MiniCard
    title="Design specifications"
    href="https://ibm.ent.box.com/folder/102045217811?s=u4acyo9oeqya8xidimxtidh8k8y08fza"
  >
    <img src={sketchSymbol} alt="Sketch"/>
  </MiniCard>
  <MiniCard
    title="Functional specifications"
    href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Card-section-images"
  >
    <img src={githubSymbol} alt="Github"/>
  </MiniCard>
  <MiniCard
    title="React Storybook"
    href="https://www.ibm.com/standards/carbon/react/?path=/story/components-card-section-images--default"
  >
    <img src={reactSymbol} alt="Storybook"/>
  </MiniCard>
  <MiniCard
    title="Web Components Storybook"
    href="https://www.ibm.com/standards/carbon/web-components/?path=/story/components-card-section-images--default"
  >
    <img src={webComponentsSymbol} alt="Storybook"/>
  </MiniCard>
</CardGroup>

> ### Content guidance

Reviewer: ```suggestion
### Content guidance for Card section with images
```

| Field                                                                           | Field type | Required | Cardinality | Max character size (English / translated) | Notes                                            |
| ------------------------------------------------------------------------------- | ---------- | -------- | ----------- | ----------------------------------------- | ------------------------------------------------ |
| Headline                                                                        | Short copy | Yes      | 1           | Mini (25 / 35)                            | –                                                |
| [Card](https://www.ibm.com/standards/carbon/components/card) | Component  | Yes      | 2–6         | –                                         | For this component, you must use the image card. |

> Note: This uses the underlying component "Card group". You can find more information about that component <a href="https://www.ibm.com/standards/carbon/components/card-group">here</a>.

Reviewer: ```suggestion
Note: This uses the underlying component [Card group](/components/card-group).

> To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.

Reviewer: ```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).
```
